jarbas/frontend/elm/Documents.elm

```diff
@@ -1,4 +1,4 @@
-module Documents exposing (Msg, Model, model, loadDocuments, getPage, update, updateLanguage, view)
+module Documents exposing (Msg, Model, model, loadDocuments, getPage, update, updateLanguage, view, totalPages, onlyDigits)
```

Comment:
As we are exposing tons of thing it might worth it to just expose everything: `module Documents exposing (..)`.